spack.yaml

```
@@ -63,7 +63,7 @@ spack:
               'SPACK_{name}_ROOT': '{prefix}'
         projections:
           all: '{name}/{version}'
-          access-om2-bgc: '{name}/2024.03.1'
+          access-om2-bgc: '{name}/2024.06.0'
           cice5: '{name}/2023.10.19'
           mom5: '{name}-bgc/2023.11.09'
```

```suggestion
          mom5: '{name}-bgc/2024.06.27'
```
Note for @CodeGat: this should have been picked up during the parsing of our `ACCESS-NRI`-specific rules, but wasn't because we take the model component name from `.spack.modules.default.tcl.include[]` and not `.spack.modules.default.tcl.projections.*`. Add an issue and reference this comment.